integrations/cross_platform/android.rst

```diff
@@ -0,0 +1,196 @@
+.. _android:
+
+|android_logo| Android
+____________________________
+
+There are several ways to cross-compile packages for `Android <https://www.android.com/>`__ platform via conan.
+
+Using android_ndk_installer
```

Comment:
make it clear that this is a Conan package. Maybe saying it is a build require directly in the title

Reply:
done